docs/Getting-Started.md

```diff
@@ -86,7 +85,20 @@ Add `PackageProviders().Append(winrt::ReactNativeWebView::ReactPackageProvider()
 
 Note if you want to enable scroll with Touch for the WebView component you must disable perspective for your app using [ReactRootView.IsPerspectiveEnabled](https://microsoft.github.io/react-native-windows/docs/ReactRootView#isperspectiveenabled).
 
-## 3. Import the webview into your component
+## 3. Decide if you want access to WebView2
+The WebView2 control is a WinUI control that renders web content using the Microsoft Edge (Chromium) rendering engine. We have added support for the WebView2 control to the WebView community module. If you want access to the WebView2 control in your app, follow these steps:
+
+  i. Ensure your React Native Windows version is v0.65 or higher.
+
+  ii. Let autolinking handle adding the `ReactNativeWebView2` project to your app.
+
+  iii. Customize your app's WinUI 2.x version to version 2.8.0-prerelease.210927001. See here for instructions.
```

Comment:
or higher; let's also file an issue for updating this when webview2 goes stable in winui2

Reply:
#2347 Filed!